packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.controller.js

```diff
@@ -0,0 +1,78 @@
+const RESCHEDULE_AUTOMATED_BACKUP_NAMESPACE =
+  'dedicated.vps.veeam.reschedule-automated-backup';
+
+export default class VpsScheduleBackupCtrl {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, Poller, VpsService) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.Poller = Poller;
+    this.VpsService = VpsService;
+    this.model = {
+      rescheduleTimepicker: null,
+    };
+  }
+
+  $onInit() {
+    this.model.rescheduleTimepicker = this.automatedBackupOption.schedule;
+    this.isLoading = false;
+  }
+
+  $onDestroy() {
+    this.Poller.kill({
+      namespace: RESCHEDULE_AUTOMATED_BACKUP_NAMESPACE,
+    });
+  }
+
+  /**
+   * Wait the triggered task
+   * @param taskId {Number}: id of running task
+   * @returns {Promise}: task promise
+   */
+  waitRescheduleAutomatedBackupTask(taskId) {
+    const endPointUrl = `/vps/${this.serviceName}/tasks/${taskId}`;
```

Comment:
Does it make sense to move this API call to Vps.service.js file?

Reply:
I don't think so @radireddy.
Do you think it should ?

Reply:
its ok if you are using it at only one place

Reply:
Yes it's.
Thanks.